src/query.rs

```diff
@@ -1656,6 +1641,118 @@ impl<'a, 'q, Q: Query> IntoIterator for &'a mut PreparedView<'q, Q> {
     }
 }
 
+/// A borrow of a [`World`](crate::World) sufficient to random-access the results of the query `Q`.
+///
+/// Note that borrows are not released until this object is dropped.
+///
+/// This struct is a thin wrapper around [`View`](crate::View). See it for more documentation.
```

Comment:
```suggestion
/// This struct is a thin wrapper around [`View`]. See it for more documentation.
```
(no-op, but we might as well be concise since we rely on `View` being in scope for all the docs below)